lib/board.rb

```diff
@@ -0,0 +1,31 @@
+class Board :cells
+
+  def initialize
+    @cells = cells
+  end
+
+  def cells
+    cells = {}
+    x_axis = ("A".."D").to_a
+    y_axis = ("1".."4").to_a
+
+    keys = []
+    x_axis.each do |x|
+      y_axis.each do |y|
+        keys << (x + y)
+      end
+    end
+
+    values = []
+    keys.each do |key|
```

Comment:
this line is repeated again below, and I'm sure there is a way to refactor so as to avoid repeating, but I can't figure it out. Let me know if you have suggestions.